### ISSUE TITLE
pixelda, the classifier (cls) is not trained on the real source dataset

### ISSUE THREAD

**Issue by SpinachR** [eriklindernoren/Keras-GAN#75]
It seems that the cls is not trained on the img_A, and it is only trained on the fake_B

**Comment by eriklindernoren** [comment]
You're right. I found that training the task network on translated images produced by the generator was enough for the task network to converge perform well on the target dataset. This is however different from the authors implementation where it's trained on the source dataset as well.

**Comment by WanyiLi** [comment]
pixelda: it seems that the Content–similarity loss is not considered.
